.github/workflows/build_sqlite_binaries.yml

```diff
@@ -0,0 +1,107 @@
+name: Build SQLite Binaries
```

Comment:
let's change file name from `.github/workflows/build_sqlite_binaries.yml` to `.github/workflows/sqlite-build.yml` so in short:
1. Use `-` dash instead of `_` underscore - easier for readability.
2. Use folder name as a first word in workflow name, so we know in which directory it works
3. `build` as action description.

Reply:
I went with underscores due to existing workflow file `update_sqlite_bindings.yml`, but I'm all for dashes 🎉

Reply:
I have updated it, thanks for pointing it out!